CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Delta Chat Android Changelog
 
+## v1.14.0
+2020-10
+
+* new swipe-to-reply option
+* disappearing messages: select for any chat the lifetime of the messages
+* chat opens at the position of the first unseen message
+* add known contacts from the IMAP-server to the local addressbook on configure
+* direct forwarding to "saved messags" - save one tap and stay in context :)
+* long tap in contact-list allows opening "profile" directly
+* allow forwarding to multiple archived chats
+* disable opportunistic encryption if unpreferred by the majority of recipients
```

Comment:
enable encryption in groups if preferred by the majority of recipients (previously, encryption was only enabled if everyone preferred it)

Reply:
yip, that's better, i changed that.